src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1786,26 +1788,28 @@ bool processOnChangeCondition
         if (condValues != NULL) {
             /* Check if some of the attributes in the NotifyCondition values list are in the entity.
              * Note that in this case we do a query for all the attributes, not restricted to attrV */
-            ContextElementResponseVector rawCerV;
+            ContextElementResponseVector rawCerV2;
```

Comment:
We have end of using the `rawCerV` variable at line 1779 with `rawCerV.release()`. Thus, I understand that the variable is "free" and can be used again here. Is that correct?